onnxruntime/core/providers/js/operators/resize.cc

```diff
@@ -26,8 +40,7 @@ namespace js {
           .InputMemoryType(OrtMemTypeCPUInput, 1)                         \
           .InputMemoryType(OrtMemTypeCPUInput, 2)                         \
           .InputMemoryType(OrtMemTypeCPUInput, 3)                         \
-          .TypeConstraint("T1", DataTypeImpl::GetTensorType<float>())     \
-          .TypeConstraint("T2", DataTypeImpl::GetTensorType<float>()),    \
```

Comment:
is there a reason to remove type constraint for T2?

Reply:
T2 is not specified for the CPU and CUDA ep. It is an optional input for opset 13 and above.

Reply:
However adding back this constraint for JSEP makes sense because currently we support float type only.